docs/generators/aspnetcore.md

```diff
@@ -24,6 +24,7 @@ These options may be applied as additional-properties (cli) or configOptions (pl
 |operationResultTask|Set methods result to Task&lt;&gt;.| |false|
 |packageAuthors|Specifies Authors property in the .NET Core project file.| |OpenAPI|
 |packageCopyright|Specifies an AssemblyCopyright for the .NET Framework global assembly attributes stored in the AssemblyInfo file.| |No Copyright|
+|packageDescription|Specifies a AssemblyDescription for the .NET Framework global assembly attributes stored in the AssemblyInfo file.| |A library generated from a OpenAPI doc|
```

Comment:
Instead of adding another CLI option, what about using `{{appDescription}}` instead ?

Ref: https://github.com/OpenAPITools/openapi-generator/blob/ba4aae5074c6d786e375887b24a38703cf7a6410/modules/openapi-generator/src/main/java/org/openapitools/codegen/DefaultGenerator.java#L296

Reply:
We could do that. The way I did it makes it consistent with the csharp-netcore generator. We might in general think about refactoring that one as well then and getting rid of duplicate options, though it might be an incompatible change so that might have to wait until 6.0.0

I could just add it in additional properties instead. Would that be better? I think having it at least somewhat consistent between generators of the same language family makes sense.

Reply:
👌 let's go with what you've so far.